### PR TITLE
feat: add lemmas about measurability and integrability of a sum

### PR DESCRIPTION
**Pull request by LorenzoLuccioli**
Add Measurable.mul_iff_right and analogous lemmas for left, and for other kinds of measurability, this part has already been PRd to mathilb.
Add integrable_add_iff_of_nonneg and auxiliary lemmas.

This PR depends on #15.

### CODE REVIEW

File: TestingLowerBounds/ForMathlib/Measurable.lean [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+@[to_additive (attr := measurability)]
+theorem AEStronglyMeasurable.mul_iff_left [TopologicalSpace β] [CommGroup β] [ContinuousMul β] [ContinuousInv β] {f g : α → β} (hf : AEStronglyMeasurable f μ) : AEStronglyMeasurable (g * f) μ ↔ AEStronglyMeasurable g μ :=
+  mul_comm g f ▸ AEStronglyMeasurable.mul_iff_right hf
+
+--TODO: put this in te right place, and PR this to mathlib, moreover write similar results for measurable, aemeasurable ecc...
+
+--maybe they can be put in the same section as:
+#check MeasureTheory.AEStronglyMeasurable.mul
+--the ones about measurability can be put in the same section as:
+#check Measurable.add
```
Comment (RemyDegenne) [inline TestingLowerBounds/ForMathlib/Measurable.lean]: That todo is already done, no?

File: TestingLowerBounds/ForMathlib/IntegrableNonneg.lean [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import TestingLowerBounds.ForMathlib.Measurable
+
+open Real
+
```
Comment (RemyDegenne) [inline TestingLowerBounds/ForMathlib/IntegrableNonneg.lean:4]:
```suggestion
```

File: TestingLowerBounds/ForMathlib/IntegrableNonneg.lean [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+namespace MeasureTheory
+
+variable {α : Type*} {mα : MeasurableSpace α} {μ ν : Measure α}
+
```
Comment (RemyDegenne) [inline TestingLowerBounds/ForMathlib/IntegrableNonneg.lean:8]:
```suggestion
```

File: TestingLowerBounds/ForMathlib/IntegrableNonneg.lean [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    (hf : f ≤ᵐ[μ] 0) (hg : g ≤ᵐ[μ] 0) :
+    Integrable (f + g) μ ↔ Integrable f μ ∧ Integrable g μ := by
+  rw [← integrable_neg_iff, ← integrable_neg_iff (f := f), ← integrable_neg_iff (f := g), neg_add]
+  exact integrable_add_iff_of_nonneg h_meas.neg (hf.mono (fun a ↦ neg_nonneg_of_nonpos))
+    (hg.mono (fun a ↦ neg_nonneg_of_nonpos))
+
+--should all of the above be calles ...of_ae_non... instead of ...of_non...?
+
+end MeasureTheory
+
```
Comment (RemyDegenne) [inline TestingLowerBounds/ForMathlib/IntegrableNonneg.lean:46]:
```suggestion
```

File: TestingLowerBounds/ForMathlib/IntegrableNonneg.lean [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+--we could put this in the same section as:
+#check MeasureTheory.Integrable.add
+--or maybe in some other file, if the file L1space is too general, since in this case we are talking about functions on ℝ
+--also, is there some way to generalize this lemma to functions on a more general space?
+#minimize_imports
```
Comment (RemyDegenne) [inline TestingLowerBounds/ForMathlib/IntegrableNonneg.lean]:
```suggestion
```

File: TestingLowerBounds/ForMathlib/IntegrableNonneg.lean [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    integrable_right_of_integrable_add_of_nonneg h_meas hf hg h⟩, fun ⟨hf, hg⟩ ↦ hf.add hg⟩
+
+lemma integrable_add_iff_of_nonpos {f g : α → ℝ} (h_meas : AEStronglyMeasurable f μ)
+    (hf : f ≤ᵐ[μ] 0) (hg : g ≤ᵐ[μ] 0) :
+    Integrable (f + g) μ ↔ Integrable f μ ∧ Integrable g μ := by
+  rw [← integrable_neg_iff, ← integrable_neg_iff (f := f), ← integrable_neg_iff (f := g), neg_add]
+  exact integrable_add_iff_of_nonneg h_meas.neg (hf.mono (fun a ↦ neg_nonneg_of_nonpos))
+    (hg.mono (fun a ↦ neg_nonneg_of_nonpos))
+
+--should all of the above be calles ...of_ae_non... instead of ...of_non...?
```
Comment (RemyDegenne) [inline TestingLowerBounds/ForMathlib/IntegrableNonneg.lean]:
```suggestion
```
It's fine to omit some `ae` for brevity.

File: TestingLowerBounds/ForMathlib/Measurable.lean (hunk 1 above)
Comment (LorenzoLuccioli) [reply TestingLowerBounds/ForMathlib/Measurable.lean]: Yes, but the PR hasn't been accepted yet

**Comment by RemyDegenne** [comment]
Could you merge master to reduce the diff and check that there is no conflict? #15 was merged.

**Review by RemyDegenne — APPROVED** [review]

**Pull request merged**